### DISCUSSION TITLE
Create New Table - Properties - Auto Increment - 0? (zero)

### DISCUSSION THREAD

**Discussion by givihuda** [dbeaver/dbeaver#20103] [Q&A]
Why is it set to zero by default?

![image](https://github.com/dbeaver/dbeaver/assets/74785256/30c93b38-fb78-4093-b607-3ea665a8492d)

Is this supposed to mean boolean "false"? Does 1 mean boolean "true"? What if I put 2?

In a MySQL CREATE statement, AUTO_INCREMENT is either there or it isn't; there's no 0 or 1 that I've ever seen.

If the field is supposed to be boolean, why not have it be a drop-down of True or False?

**Comment by givihuda** [comment]
Also, I'm seeing no documentation about this.

**Comment by LonwoLonwo** [comment]
Hello @givihuda 

This is "The next AUTO_INCREMENT value.". DBeaver reads it from SHOW TABLE STATUS. Here is the documentation: https://dev.mysql.com/doc/refman/8.0/en/show-table-status.html

A description for the property will be added in the next release 23.1.0.

![2023-05-29 18_54_47-DBeaver Ultimate 23 1 0 - penguins](https://github.com/dbeaver/dbeaver/assets/45152336/18b2880d-e897-450a-9121-4209fe72af63)
